### PR TITLE
Bs request query

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

File: src/api/app/models/bs_request.rb [hunk 1]
```diff
@@ -33,7 +33,7 @@ class SaveError < APIException
   scope :to_accept, -> { where(state: 'new').where('accept_at < ?', DateTime.now) }
   # Scopes for collections
   scope :with_actions, -> { joins(:bs_request_actions).distinct.order(priority: :asc, id: :desc) }
-  scope :in_states, ->(states) { where('bs_requests.state in (?)', states).references(:bs_requests) }
+  scope :in_states, ->(states) { where('bs_requests.state in (?)', states) }
```
Comment (mdeniz) [inline src/api/app/models/bs_request.rb]: It can be also like this?

```
scope :in_states, ->(states) { where(state: states) }
```

Comment (bgeuken) [inline src/api/app/models/bs_request.rb]: I guess so. But I would say this is out of the scope of this PR. We can change this later on.

Comment (bgeuken) [inline src/api/app/models/bs_request.rb]: Alright. That seems to work and I've added the patch now.

**Comment by mdeniz** [comment]
LGTM

**Pull request merged**